lib/Disco.php

```diff
@@ -970,4 +977,10 @@ private function fillSpName($t)
             //OK, we will just display the disco
         }
     }
+
+    private function prepareAcrsForMfa(array &$state)
+    {
+        $contextsToAdd = $this->wayfConfiguration->getArray(self::ADD_AUTHN_CONTEXT_CLASSES_FOR_MFA, []);
+        Module\perun\Auth\Process\RestoreAcrs::storeAcrs($state, $contextsToAdd);
```

Comment:
```suggestion
        RestoreAcrs::storeAcrs($state, $contextsToAdd);
```
+ add `use SimpleSAML\Module\perun\Auth\Process\RestoreAcrs;` at the top